src/unit_tests/wazuh_db/test_wdb_global.c

```diff
@@ -718,6 +718,72 @@ void test_wdb_global_sync_agent_info_get_full(void **state)
     assert_int_equal(result, WDBC_DUE);
 }
 
+void test_wdb_global_sync_agent_info_get_size_limit(void **state)
+{
+    int result = 0;
+    int last_agent_id = 0;
+    test_struct_t *data  = (test_struct_t *)*state;
+    char *output = NULL;
+    cJSON *json_output = NULL;
+    cJSON *root = NULL;
+    cJSON *json_agent = NULL;
+    int agent_id = 10000;
+
+    root = cJSON_CreateArray();
+    json_agent = cJSON_CreateObject();
+    cJSON_AddNumberToObject(json_agent, "id", agent_id);
+    // Creating a cJSON array of WDB_MAX_RESPONSE_SIZE
```

Comment:
Maybe we can use the `WDB_MAX_RESPONSE_SIZE` value to create the test command, and not a fixed value, just in case it changes in the future. This would keep the test valid.

Reply:
As the test requires a valid json string to be parsed into the tested method, it's not possible to create a string with the specific size. This is why the json is built in this way, with specific values that finally end in the json length required.

Reply:
You are right!